### ISSUE TITLE
notify creator on blocked comment email: link to discussion leads to module detail

### ISSUE THREAD

**Issue by Rineee** [liqd/a4-kosmo#244]
**URL:** 
**user:** 
**expected behaviour:** the links 'zur Diskussion' leads me to the discussion 
**behaviour:** link leads to module detail
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?

**Comment by fuzzylogic2000** [comment]
Right! I thought, this was about a project/module timeline mixup. But it is linking to the right page, but not the comments. As far as I know, we can link to a comment, but not the discussion itself? @phillimorland is that right?
We did not link to the blocked comment itself on purpose here, so I don't really know what to do about it. I mean, we could link to that comment and mean the discussion around it by the wording. Or we could change the wording.

**Comment by phillimorland** [comment]
@fuzzylogic2000 @Rineee if we want it to scroll to the comments we can add #textarea-top to the url, that's the id of the comment textarea, we use it in the debate module

**Comment by fuzzylogic2000** [comment]
> @fuzzylogic2000 @Rineee if we want it to scroll to the comments we can add #textarea-top to the url, that's the id of the comment textarea, we use it in the debate module

Nice, thank you!

**Comment by Rineee** [comment]
@fuzzylogic2000 No, I meant it is linking to the wrong page.. in the e-mail I got, it links to the idea overview page and I thought it should link to the idea detail at least, where the comment belonged to? Right?

**Comment by fuzzylogic2000** [comment]
> @fuzzylogic2000 No, I meant it is linking to the wrong page.. in the e-mail I got, it links to the idea overview page and I thought it should link to the idea detail at least, where the comment belonged to? Right?

Ah, yes! I can fix that! :)
